frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -977,9 +978,9 @@ export const transformKnNodeData = (
                 metadata: {
                   uid: sinkTargetUid,
                   namespace: data.resources.obj.metadata.namespace || '',
+                  annotations: { [typeAnnotation]: NodeType.SinkUri },
                 },
                 spec: { sinkUri },
-                type: { nodeType: NodeType.SinkUri },
                 kind: 'URI',
```

Comment:
Can we not use the fake `kind` we created here instead of a new annotation?

Reply:
The concern was to keep it specific to SinkUri. So we chose to use type `sink-uri` and annotaion seemed to be the proper place to use it.
cc @invincibleJai

Reply:
@nemesis09 we can go with `kind` as suggested, second thought was if in future any resource has kind `URI` which is not the case so far

Reply:
updated to use kind

Reply:
Since this is a made up resource, we could make a more unique kind as well and even pair it with a unique api group.